config/manager/kustomization.yaml

```diff
@@ -11,11 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-namespace: default
-
+---
 resources:
-- operator.yaml
```

Comment:
The indentation is different. Is this deliberate?

Reply:
Must have got formatted when I added the `patchesStrategicMerge`